Dockerfile.ci

```diff
@@ -1107,7 +1107,7 @@ ARG AIRFLOW_CONSTRAINTS_LOCATION=""
 ARG DEFAULT_CONSTRAINTS_BRANCH="constraints-main"
 # By changing the epoch we can force reinstalling Airflow and pip all dependencies
 # It can also be overwritten manually by setting the AIRFLOW_CI_BUILD_EPOCH environment variable.
-ARG AIRFLOW_CI_BUILD_EPOCH="10"
+ARG AIRFLOW_CI_BUILD_EPOCH="11"
```

Comment:
```suggestion
ARG AIRFLOW_CI_BUILD_EPOCH="10"
```

Reply:
Since markdown is transitive dependency, we can get rid of EPOCH changing (Cache will remain untouched)